indico/modules/events/papers/views.py

```diff
@@ -16,22 +16,28 @@
 
 from __future__ import unicode_literals
 
+from indico.modules.events.abstracts.views import _MathjaxMixin
```

Comment:
which events/util folder? also, not a fan of a base.py that's not for base classes